docs/product-manuals/zeebe/open-source/deprecated-features.md

```diff
@@ -32,3 +32,7 @@ In terms of specifying values, there were two minor changes:
 
 - Memory sizes are now specified like this: `512MB` (old way: `512M`)
 - Durations, e.g. timeouts, can now also be given in ISO-8601 Durations format. However you can still use the established way and specify a timeout of `30s`
+
+## Removed in 1.0
```

Comment:
Being a bit nit-picky here: Since 1.0 is not released yet this currently describes the future. So we could either remove, or rephrase it to make it clearer that it will be removed.